chromadb/ingest/__init__.py

```diff
@@ -1,8 +1,37 @@
 from abc import ABC, abstractmethod
 from typing import Callable, Optional, Sequence
 from overrides import EnforceOverrides
-from chromadb.types import InsertEmbeddingRecord, EmbeddingRecord, SeqId
+from chromadb.types import (
+    SubmitEmbeddingRecord,
+    EmbeddingRecord,
+    SeqId,
+    Vector,
+    ScalarEncoding,
+)
 from uuid import UUID
+import array
+
+
+def encode_vector(vector: Vector, encoding: ScalarEncoding) -> bytes:
+    """Encode a vector into a byte array."""
+
+    if encoding == ScalarEncoding.FLOAT32:
+        return array.array("f", vector).tobytes()
```

Comment:
is this platform safe in python? i.e could you ship a file that saves these between platforms and have it encode/decode OK?